py-polars/polars/expr/expr.py

```diff
@@ -5005,9 +5005,11 @@ def eq(self, other: Any) -> Self:
 
     def eq_missing(self, other: Any) -> Self:
         """
-        Method equivalent of equality operator `expr == other` where `None == None`.
+        Equality operator where `None` is treated as a distinct value.
 
         This differs from default `eq` where null values are propagated.
+        With this method, `None == None` returns `True` instead of `None`,
+        and `x == None` returns `False` instead of `None` where `x` is not `None`.
```

Comment:
This is honestly not really clear for me.

Reply:
I also don't think the example should be using the `==` sign since that is an explit operator for the `eq` method. How about:

> Equality operator where null is treated as a distinct value.
>
> With this method, null is equal to null and is not equal to any other value.